README.md

```diff
@@ -87,8 +87,15 @@ spec:
 ## Quick Start
 
 Clone the repository and run `make local` in an existing `kube:admin` OpenShift
-CLI session. 
+CLI session
 
+### Running operator locally
+
+Clone the repository and run `make local` in an existing `kube:admin` OpenShift CLI session.
```

Comment:
This sentence is the same as the one above under "Quick Start" - should we remove the one from above?

Reply:
Done